modules/nf-core/agat/spaddintrons/tests/main.nf.test

```diff
@@ -0,0 +1,60 @@
+nextflow_process {
+
+    name "Test Process AGAT_SPADDINTRONS"
+    script "../main.nf"
+    process "AGAT_SPADDINTRONS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "agat"
+    tag "agat/spaddintrons"
+
+    test("homo_sapiens - gtf") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    file(params.test_data['homo_sapiens']['genome']['genome_gtf'], checkIfExists: true)
+                ]
+                input[1] = []
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("homo_sapiens - gtf - stub") {
+
+        options "-stub"
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    file(params.test_data['homo_sapiens']['genome']['genome_gtf'], checkIfExists: true)
+                ]
+                input[1] = []
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
```

Comment:
```suggestion
                { assert snapshot(process.out.gff.collect { file(it[1]).getName() } +
                                process.out.versions ).match() }
```

Reply:
rather than capturing md5sum of empty file, what do you think about capturing only the file name in the snap for the stub ?

Reply:
not a deal breaker as the name is recorded anyway...upto you :smile:

Reply:
i can change it. but honestly i struggle with the syntax for special cases like this, so i have a tendency to use the simplest syntax possible and just proceed if it works.

Reply:
need to update snap